lightning/src/ln/channel_state.rs

```diff
@@ -363,6 +369,7 @@ pub struct ChannelDetails {
 	/// This does not consider any on-chain fees.
 	///
 	/// See also [`ChannelDetails::outbound_capacity_msat`]
+	#[deprecated(since = "0.0.124", note = "use [`ChannelMonitor::get_claimable_balances`] instead")]
```

Comment:
Eventually this intradoc link will work (with some updating): https://github.com/rust-lang/rust/issues/98342